application/forms/Command/Object/ScheduleServiceDowntimeForm.php

```diff
@@ -19,8 +20,47 @@ class ScheduleServiceDowntimeForm extends CommandForm
 {
     use Auth;
 
+    /** @var DateTime downtime start */
+    protected $start;
+
+    /** @var DateTime fixed downtime end */
+    protected $fixedEnd;
+
+    /**@var DateTime flexible downtime end */
+    protected $flexibleEnd;
+
+    /**  @var DateInterval flexible downtime duration */
+    protected $flexibleDuration;
+
+    /** @var mixed Comment Text */
+    protected $commentText;
+
+    /**
+     * Initialize this form
+     */
+    public function __construct()
+    {
+        $this->start = new DateTime();
+
+        $config = Config::module('icingadb');
+
+        $this->commentText =  $config->get('settings', 'hostdowntime_comment_text');
```

Comment:
Please only a single space after operators. And this should the service downtime comment!